staging/src/k8s.io/client-go/util/workqueue/queue_test.go

```diff
@@ -321,3 +323,40 @@ func TestQueueDrainageUsingShutDownWithDrainWithDirtyItem(t *testing.T) {
 
 	finishedWG.Wait()
 }
+
+type bigObject struct {
+	data []byte
+}
+
+// Created outside TestKubeLeak so it won't be GCed.
```

Comment:
should be able to reference from inside the registered Cleanup closure to keep it from going out of scope, without making this a stateful package var, right?